packages/venia-ui/lib/components/MagentoRoute/magentoRoute.js

```diff
@@ -7,17 +7,18 @@ import {
 } from '@magento/peregrine/lib/talons/MagentoRoute';
 
 import { fullPageLoadingIndicator } from '../LoadingIndicator';
+import { useIntl } from 'react-intl';
```

Comment:
Third party deps should be near the top of the imports.

Can you make sure this change is applied throughout the changeset?